proposals/wac-ucr/index.bs

```diff
@@ -911,6 +911,21 @@ write access. Alice grants *PerformChart* read only access to all the projects
 that she can access.
 
 
+### Application discovering access privileges {#uc-client-discover-access-privileges}
+
+Guinan uses an application to author and publish documents. The application
+adapts its UI to the access privileges in order to distinguish between allowed
+(actionable) and disallowed features for different permission groups. The
+application can make use of information to assist Guinan based on what it is
+allowed to perform, the public, as well as any other group.
```

Comment:
The last sentence of the paragraph above, especially the last line of the block (920), needs rewording. I don't understand what it's trying to say, so I cannot offer any suggestion.

Reply:
Thanks. I agree. I gave it another pass. What do you think?